src/core/client_channel/backup_poller.cc

```diff
@@ -62,15 +62,16 @@ static grpc_core::Duration g_poll_interval =
 static bool g_backup_polling_disabled;
 
 void grpc_client_channel_global_init_backup_polling() {
-#ifndef GRPC_DO_NOT_INSTANTIATE_POSIX_POLLER
+#ifdef GRPC_DO_NOT_INSTANTIATE_POSIX_POLLER
```

Comment:
nit: consider renaming this macro to `GRPC_BUILD_FOR_PYTHON` to make the intention clear